src/ReactDots.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+export default class Dots extends React.Component {
+  constructor (props) {
+    super(props)
+
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  handleClick (nbr) {
+    this.props.handleClickDot(nbr)
+  }
+
+  render () {
+    const { nbrSlides, currentSlide } = this.props
+    const items = []
+
+    for (let i = 0; i < nbrSlides; i++) {
+      items.push(
+        <li className={'Dot' + (i === currentSlide ? ' current' : '')} onClick={_ => this.handleClick(i)} key={i}>&nbsp;</li>
```

Comment:
key === index là
pourquoi tu ne map pas ici direct dans le jsx ? 
pas de reco particulière, si ce nest d'avoir la meme consistance sur l'ensemble du proj

Reply:
Je trouve le return plus lisible comme ça, that's all.